### DISCUSSION TITLE
How to use html-to-markdown?

### DISCUSSION THREAD

**Discussion by bob-dpi** [thephpleague/html-to-markdown#225] [General]
This package is great for my project of converting a web site to a github docs repository, but I'm a little lost on how to use it.  Is this a stand-alone PHP script?  Is is a filter to be added to the web server PHP configuration?   Assume I'm an idiot and and I start with the shell command: 
  git clone https://github.com/thephpleague/html-to-markdown.git

What do I do next?

thanks

**Comment by colinodell** [comment]
Hi @bob-dpi,

This library needs to be installed with [Composer](https://phptherightway.com/#composer_and_packagist).  Once you've installed Composer, follow [the instructions in the README](https://github.com/thephpleague/html-to-markdown#how-to-use-it) to install this library and start using it.
